ext/NormalizingFlowsEnzymeExt.jl

```diff
@@ -0,0 +1,27 @@
+module NormalizingFlowsEnzymeExt
+
+if isdefined(Base, :get_extension)
+    using ADTypes
```

Comment:
One should only import the package itself (NormalizingFlows) and the weak dependency (Enzyme), no other packages. Not even standard libraries. They should be loaded eg indirectly via the main package.